test/OpenSpaceToolkit/Astrodynamics/Access/Generator.test.cpp

```diff
@@ -1103,6 +1105,68 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Access_Generator, ComputeAccesses_5)
     }
 }
 
+TEST_F(OpenSpaceToolkit_Astrodynamics_Access_Generator, ComputeAccesses_6)
+{
+    // Ensure no interpolation is performed outside of the analysis interval
+
+    {
```

Comment:
This test only tests exercise the upper bound code path? Or does it exercise both the upper and lower?

If its not much work (idk if you have to "know" what the accesses look like ahead of time to trigger the lower bound condition) it would be good to exercise the other code path as well

Reply:
Hashed this out in person and we agree that only the upper bound edge case is hit.